zstacklib/zstacklib/utils/file_metadata_handler.py

```
@@ -0,0 +1,490 @@
+
+    def _do_cleanup(self, metadataPath):
+        _validate_metadata_path(metadataPath)
+        with self._get_path_lock(metadataPath):
+            removed_any = False
+            for path in [metadataPath, metadataPath + '.tmp',
+                         metadataPath + '.summary', metadataPath + '.summary.tmp']:
+                try:
+                    if os.path.exists(path):
+                        os.remove(path)
+                        removed_any = True
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)